src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
This file may not be required as it doesn't look like it belongs in the project. I suggest removing it to avoid confusion in the future 😃